reconstruction/point2D.cc

```diff
@@ -1,5 +1,8 @@
 // Author: Philipp Lindenberger (Phil26AT)
 
+// Use Eigens aligned allocator for vectors
+#include<Eigen/StdVector>
```

Comment:
Code style: extra space after the include?

Reply:
Will add clang-format.